client/src/components/books/books/BooksList.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import Loader from 'halogen/ScaleLoader';
+import PropTypes from 'prop-types';
+import Button from '../../form/Button';
+
+const $ = window.$;
+/**
+ * @class BooksList
+ * @extends {Component}
+ */
+/* eslint-disable class-methods-use-this, require-jsdoc */
+class BooksList extends Component {
+  goToEditPage(event) {
+    event.preventDefault();
+    const $parent = $(event.target).parent();
+    this.context.router.push($parent.attr('to'));
+  }
+
+  confirmDelete(event) {
+    event.preventDefault();
+    const $parent = $(event.target).parent();
+    this.context.router.push($parent.attr('to'));
+  }
+
+
+  render() {
+    const style = {
+      display: '-webkit-flex',
+      display: 'flex', //eslint-disable-line
+      WebkitFlex: '0 1 auto',
+      flex: '0 1 auto',
+      WebkitFlexDirection: 'column',
+      flexDirection: 'column',
+      WebkitFlexGrow: 1,
+      flexGrow: 1,
+      WebkitFlexShrink: 0,
+      flexShrink: 0,
+      WebkitFlexBasis: '25%',
+      flexBasis: '25%',
+      width: '100%',
+      height: '100%',
+      WebkitAlignItems: 'center',
+      alignItems: 'center',
+      WebkitJustifyContent: 'center',
+      justifyContent: 'center',
+      position: 'fixed',
+      top: 0,
+      left: 0,
+      bottom: 0,
+      right: 0,
+      margin: 'auto',
+      backgroundColor: '#fff'
+    };
```

Comment:
bad practise. create a different stylesheet for this and use a class where appropriate.